tests/web/rpc/v1/test_endpoint_get_trace.py

```diff
@@ -457,3 +474,59 @@ def get_span_id(span: TraceItem) -> str:
     )[
         2:
     ].rjust(16, "0")
+
+
+@pytest.mark.clickhouse_db
+@pytest.mark.redis_db
+class TestGetTracePagination(BaseApiTest):
```

Comment:
Feels like we're missing a test case where we verify that `ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS` is honored